main.js

```diff
@@ -41,6 +41,8 @@ define(function (require, exports, module) {
 
     // Load initial document
     var defaultHTML = brackets.getModule("text!filesystem/impls/filer/lib/default.html");
+    var defaultCSS  = brackets.getModule("text!filesystem/impls/filer/lib/style.css");
+    var defaultJS   = brackets.getModule("text!filesystem/impls/filer/lib/script.js");
```

Comment:
It's because they're in this file path `filesystem/impls/filer/lib/`. Should I move them?